docs/_guide/styles.md

```diff
@@ -8,141 +8,101 @@ slug: styles
 * ToC
 {:toc}
 
-This page is a guide to the styling options available for LitElement components. 
+Add styles to your element using a static `styles` property:
```

Comment:
I'm torn on this. It is repetitive, but when we didn't have this up front, people felt there should be a simple example up front instead of just text.

@jbingham WDYT? I think you were a proponent of having the example up front.

Reply:
Merging the two sections seems good, as a way to avoid repetition. The caveat is that it would be really good to have as few words as possible before the example, since most people won't read below the fold. Eg, "This page is a public guide to the various ways to style LitElement components. TL;DR: In most cases, you should add styles to your element using the static `styles` property like this:"

Reply:
Removed the first example.